{"name": "Тест ошибки при авторизации курьера с неверным паролем", "status": "passed", "start": 1706636493394, "stop": 1706636493665, "uuid": "b36a1fc0-7a8f-4931-93c1-c296108d75fe", "historyId": "1bf0045128739c32d06e8c1f6b1ecfd4", "testCaseId": "1bf0045128739c32d06e8c1f6b1ecfd4", "fullName": "tests.test_login_courier.TestLoginCourier#test_login_courier_wrong_password_error", "labels": [{"name": "story", "value": "Тесты на логин курьера"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "Asus"}, {"name": "thread", "value": "14252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}